src/bijectors/named_bijector.jl

```diff
@@ -171,10 +171,10 @@ end
     for i = 2:length(T.parameters)
         temp = gensym(:temp)
         push!(expr.args, :($temp = forward(cb.bs[$i], y)))
```

Comment:
Another one above and here:
```suggestion
        temp = gensym(:res_logjac)
        push!(expr.args, :(y, $temp = with_logabsdet_jacobian(cb.bs[$i], y)))
```